### PR TITLE
Github.io docs changes

### PR DESCRIPTION
**Pull request by adarshan-intel**
1. Removed fetching time from google
2. Adding 1.1.7 version in main docs page
3. Fixing all inactive and broken links 
4. Changing to java 17 from java 11
5. Changing docker compose version to 1.29.2
6. Updating to openjdk 17 from openjdk 11
7. Adding utility to fetch time by ntp in RHEL

### CODE REVIEW

File: docs/community.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Welcome to the FIDO Device Onboard community. FIDO Device Onboard is an open source project sponsored by [LF Edge](https://www.lfedge.org/projects/fidodeviceonboard/).
 
-Community practices are documented at the [Secure Device Onboard wiki](https://wiki.lfedge.org/display/SDO/Secure+Device+Onboard).
+Community practices are documented at the [Secure Device Onboard wiki](https://wiki.lfedge.org/display/LE/Secure+Device+Onboard).
```
Comment (DukeDavis12) [inline docs/community.md]: Replace it with the respective FDO link.

File: docs/implementation-references/getting-started-guide.md [hunk 2]
```diff
@@ -274,7 +274,7 @@ sudo docker-compose up --build
 Once the Owner instance has successfully started, the following output is displayed
 <figure>
  <center>
-  <img src="../../images/owner.png"/>
+  <img src="../images/owner.png"/>
```
Comment (DukeDavis12) [inline docs/implementation-references/getting-started-guide.md:288]: Whether img src changes required ?

File: homepage/docs/index.md [hunk 3]
```diff
@@ -6,6 +6,7 @@ implementations for FIDO Device Onboard specification.
 Following snapshots are available for different releases of FIDO Device Onboard implementations.
 
 * [latest](https://fido-device-onboard.github.io/docs-fidoiot/latest)
+* [1.1.7](https://fido-device-onboard.github.io/docs-fidoiot/latest)
```
Comment (DukeDavis12) [inline homepage/docs/index.md]:
```suggestion
* [1.1.7](https://fido-device-onboard.github.io/docs-fidoiot/1.1.7)
```

File: docs/implementation-references/getting-started-guide.md (hunk 2 above)
Comment (adarshan-intel) [reply docs/implementation-references/getting-started-guide.md:288]: Yes this is the correct path

File: docs/community.md (hunk 1 above)
Comment (adarshan-intel) [reply docs/community.md]: https://github.com/fido-device-onboard/docs-fidoiot/pull/63/commits/cde3715f015092abe999b2fc63b1936d78b2e939

File: docs/implementation-references/getting-started-guide.md (hunk 2 above)
Comment (adarshan-intel) [reply docs/implementation-references/getting-started-guide.md:288]: Yes this is the correct path

File: homepage/docs/index.md (hunk 3 above)
Comment (adarshan-intel) [reply homepage/docs/index.md:9]: https://github.com/fido-device-onboard/docs-fidoiot/pull/63/commits/b557cec762e26751df3efa35ed9e58b10ae04662

File: docs/community.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 
 Welcome to the FIDO Device Onboard community. FIDO Device Onboard is an open source project sponsored by [LF Edge](https://www.lfedge.org/projects/fidodeviceonboard/).
 
-Community practices are documented at the [Secure Device Onboard wiki](https://wiki.lfedge.org/display/SDO/Secure+Device+Onboard).
+Community practices are documented at the [Fido Device Onboard wiki](https://www.lfedge.org/projects/fidodeviceonboard).
```
Comment (Sai-Anudeep47) [inline docs/community.md]:
```suggestion
Community practices are documented at the [FIDO Device Onboard wiki](https://www.lfedge.org/projects/fidodeviceonboard).
```

Comment (adarshan-intel) [reply docs/community.md]: https://github.com/fido-device-onboard/docs-fidoiot/pull/63/commits/1a4e4f9b2e98de023cd0bbcc2ba9541318d41fbf

**Review by Sai-Anudeep47 — APPROVED** [review]

**Review by DukeDavis12 — APPROVED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

**Pull request merged**